app/models/ability.rb

```diff
@@ -0,0 +1,12 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    can :read, Recipe, public: true
+    can :read, Food
+    return unless user.present?
+
+    can :manage, Recipe, user: user
+    can :manage, Food, user:
```

Comment:
- [ ] This statement is not complete and seems to throw an error on start up

Reply:
This one I blame on rubocop.